packages/react/test/specs/components/Attachment/Attachment-test.tsx

```diff
@@ -18,5 +21,32 @@ describe('Attachment', () => {
     handlesAccessibility(Attachment, {
       defaultRootRole: undefined,
     })
+
+    test('handleClick is executed when Enter is pressed for attachment, not for more option button', () => {
+      const onClickAttachment = jest.fn()
+      const onClickButton = jest.fn()
+      const attachement = mountWithProvider(
+        <Attachment
+          actionable
+          action={{
+            icon: 'more',
+            onClick: () => onClickButton(),
+          }}
+          onClick={onClickAttachment}
+        />,
+      )
+      const moreOptionButton = attachement.find('button')
+
+      attachement.simulate('keydown', { keyCode: keyboardKey.Enter })
```

Comment:
maybe get the actual DOM element and simulate a keydown on it? instead of `attachment`, use `attachment.find(Attachment.className).simulate(...)`

Reply:
why it is different, if I use attachment object which is created, or if I will find the object in DOM?
I took this approach from "ListItem-test.tsx" where it was done some time ago. 
If there is some benefit getting object from DOM, then we should consider do the change across the tests.